test/source/tests/tests/compose.ts

```diff
@@ -541,6 +542,19 @@ export const defineComposeTests = (testVariant: TestVariant, testWithBrowser: Te
       expect(await body.$eval('#input_text img', el => el.getAttribute('src'))).to.eq(imgBase64);
     }));
 
+    ava.default('compose - saving and rendering a draft when offline', testWithBrowser('compatibility', async (t, browser) => {
+      let composePage = await ComposePageRecipe.openStandalone(t, browser, 'compatibility');
+      await (composePage.target as Page).setOfflineMode(true); // go offline mode
+      await ComposePageRecipe.fillMsg(composePage, { to: 'human@flowcrypt.com' }, 'offline test', {});
+      expect(await composePage.read('@action-send')).to.eq('Re-enter recipient..'); // ensure offline mode
+      await composePage.type('@input-body', `This is a test of saving a draft when offline`);
+      await ComposePageRecipe.waitWhenDraftIsSaved(composePage);
```

Comment:
Here could test for presense of the different "offline" draft saved note, proposed above

Reply:
Done :+1: